backend/services/implementations/__tests__/userService.test.ts

```diff
@@ -90,4 +90,37 @@ describe("mongo userService", (): void => {
 
     expect(emptyRes.length).toEqual(0);
   });
+
+  it("getAllTeachers", async () => {
+    const createRes = await UserModel.insertMany(testUsers);
+
+    const testSchools = [
+      {
+        name: "school1",
+        country: "some-country",
+        subRegion: "some-region1",
+        city: "some-city",
+        address: "some-address",
+        teachers: [createRes[0].id],
+      },
+      {
+        name: "school2",
+        country: "some-country",
+        subRegion: "some-region2",
+        city: "some-city",
+        address: "some-address",
+        teachers: [createRes[1].id],
+      },
+    ];
+
+    await SchoolModel.insertMany(testSchools);
+    const res = await userService.getAllTeachers();
+
+    expect(res[0].school).toEqual("school2");
```

Comment:
can you follow the format of the other tests for consistency? i.e. forEach loop and don't hardcode the expected values